src/ImageSharp/Formats/Jpg/Components/MutableSpan.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Buffers;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace ImageSharp.Formats
+{
+    /// <summary>
+    /// Like corefxlab Span, but with an AddOffset() method for efficiency.
+    /// TODO: When Span will be official, consider replacing this class!
```

Comment:
Hopefully. However, the main purpose would be to have a code more conform to the style of the new fancy super-fast .NET API-s.